generate_scripts.py

```diff
@@ -30,26 +28,37 @@ def render(file_prefix, template_file, browser_executable):
                 navigation_cycles=30, 
                 per_navigation_delay=15, 
                 delay=3600, 
-                browser=browser_executable))
+                browser=process_name))
 
-for template_file in ['open_background', 'idle_on_site', 'idle', 'scroll', 'navigation', 'alligned_timers', 'zero_window']:
 
-    with open("./driver_scripts_templates/"+template_file) as file_:
-        template = Template(file_.read())
-        
-        # Generate for all Chromium based browsers
-        for browser in ['Chrome', 'Canary', "Chromium", "Edge"]:
-            browser_executable = utils.browsers_definition[browser]["executable"]
-            render(browser, template_file, browser_executable)
+def render_runner_scripts():
+    for template_file in ['open_background', 'idle_on_site', 'idle', 'scroll', 'navigation', 'alligned_timers', 'zero_window']:
 
-    # Skip alligned timer case as chrome only
-    if template_file == "alligned_timers":
-        continue
+        with open("./driver_scripts_templates/"+template_file) as file_:
+            template = Template(file_.read())
+            
+            # Generate for all Chromium based browsers
+            for browser in ['Chrome', 'Canary', "Chromium", "Edge"]:
+                process_name = utils.browsers_definition[browser]["process_name"]
+                render(browser, template, template_file, process_name)
 
-    template_file = "safari_"+template_file
-    with open("./driver_scripts_templates/"+template_file) as file_:
-        template = Template(file_.read())
+        # Skip alligned timer case as chrome only
+        if template_file == "alligned_timers":
+            continue
 
-        # Generate for Safari
-        render("", template_file, "")
+        template_file = "safari_"+template_file
+        with open("./driver_scripts_templates/"+template_file) as file_:
+            template = Template(file_.read())
+
+            # Generate for Safari
+            render("", template, template_file, "")
         
+def generate_all():
+  shutil.rmtree("driver_scripts/", ignore_errors=True)
+  os.makedirs("driver_scripts", exist_ok=True)
+  render_runner_scripts()
+
+  # Copy the files that don't need any substitutions. 
+  for script in ["idle", "prep_safari", "finder"]:
```

Comment:
driver_scripts_templates/idle is different from driver_scripts/idle as it still expects a browser. How do we handle the no browser idle script?

Reply:
I'm not sure what you mean that it differs? It's listed in the pull request as not having changed.

Reply:
Discussed offline. There's idle and idle.scpt.
We should keep only idle.scpt.